content/net/sourceforge/pmd/pmd-6.42.0.buildspec

```diff
@@ -0,0 +1,16 @@
+groupId=net.sourceforge.pmd
+artifactId=pmd
+display=${groupId}:${artifactId}
+version=6.42.0
+
+gitRepo=https://github.com/pmd/${artifactId}.git
+gitTag=pmd_releases/${version}
+
+tool=mvn
+jdk=11
+newline=lf
+
+command="mvn -Ppmd-release clean package -DskipTests -Dmaven.javadoc.skip -Dgpg.skip"
```

Comment:
we actually use maven wrapper - so would this be more appropriate in terms of reproducibility?

```suggestion
command="./mvnw -Ppmd-release clean package -DskipTests -Dmaven.javadoc.skip -Dgpg.skip"
```

Reply:
good idea, yes, to easily match your env in a reproducible way